proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -86,7 +86,6 @@ message GetMutableStateResponse {
     // (-- api-linter: core::0140::prepositions=disabled
     //     aip.dev/not-precedent: "to" is used to indicate interval. --)
     google.protobuf.Duration sticky_task_queue_schedule_to_start_timeout = 11 [(gogoproto.stdduration) = true];
-    int32 event_store_version = 12;
```

Comment:
(here and in all other `.proto` field removals) This looks good, but I would vote for doing something along the lines of 

```
    int32 OBSOLETE_event_store_version = 12 [deprecated = true];
```
or 
```
    reserved 12
```

This would answer possible future questions of "hey what happened to # 12?" and make it less likely that this slot gets reused (somehow) causing mis-deserialization.
 
https://developers.google.com/protocol-buffers/docs/proto3#updating